doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+in the frequency range, the frequency-sweep Krylov method performs the following steps to approximate the response 
+across the frequency range :
+
+* Builds a Krylov subspace set of vectors at the frequency value in the middle of the requested frequency range,
+* Reduces the system matrices and loading on the entire frequency range,
+* Solves the reduced system, and
+* Expands the results back to compute the harmonic response. 
+
+MAPDL provides the following methods to implement a harmonic analysis using the Krylov Method:
+
+#. Via Mechanical APDL commands.
+#. Via macros, a feature described in *Frequency-Sweep Harmonic Analysis via the Krylov Method* in the *Structural Analysis Guide*.
+#. PyMAPDL implementation of the macros, a feature described here.
+
+Assumptions
+-----------
+The following assumptions are made in the Krylov solution obtained using this method :
+
+* The stiffness, mass, and damping matrices are assumed to be constant (independent of frequency).
+
+* The external load vector is linearly ramped over frequency. Ramping assumes that the frequency at 
+  which the Krylov subspace is built is in the middle of the frequency range. If you wish to apply 
+  stepped loading, there is an option to specify that in the inputs for the 
+  :func:`KrylovSolver.krysolve() <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` method.
+
+
+Krylov : PyMAPDL implementation
+===============================
+The PyMAPDL implementation gives the user customization and flexibility in terms of accessing
+the subspace vectors and reduced solutions in the python format for defining user defined routines.
+
+If you do not require customization, you can use the simpler using APDL commands to solve a harmonic analysis 
+with the Krylov method.
+ 
+The procedure described here implements an analysis identical to that defined by the macros.
+Details on the theory and equations describing the Krylov method can be found in the works of Puri [1]_ and Eser [2]_ .
+
+The exposure in PyMAPDL follows the same the same theory as the apdl macro and has the following methods:
+
+* :func:`KrylovSolver.krygensub() <ansys.mapdl.core.krylov.KrylovSolver.krygensub>` Creates the Krylov subspace. 
+* :func:`KrylovSolver.krysolve() <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` Solves the reduced system of equations.
+* :func:`KrylovSolver.kryexpand() <ansys.mapdl.core.krylov.KrylovSolver.kryexpand>` Expands the Krylov subspace.
+
+.. warning:: These methods must be run consecutively.
+
+Usage
+=====
+
+Generate the full file (``.full``) for Krylov method using Mechanical APDL:
+
+.. code:: py
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    
+    >>> mapdl = launch_mapdl()
+    >>> mapdl.prep7()
+
+	# Generate the FEA model (mesh, constraints, loads, etc.)
+  # ...
+
+    >>> mapdl.run("/SOLU")
+    >>> mapdl.antype("HARMIC")  # HARMONIC ANALYSIS
+    >>> mapdl.hropt("KRYLOV")
+    >>> mapdl.eqslv("SPARSE")
```

Comment:
There is a command to turn off distributed processing, you might consider it to add it here.

I dont remember the name of the command though.

Reply:
Forget about this, I had a conversation with @mikerife  and he didn't recommend to use it because the command (btw it is ``RESWRITE``) does append the results, not write them.